api/tests/roomTests.js

```diff
@@ -101,7 +106,50 @@ describe('/rest/rooms', async () => {
     it('should return the correct status and error when passed a valid uuid that does not exist', async () => {
       const randomId = uuidv4();
       const result = await agent
-        .put(`/rest/rooms/${randomId}/join/${testUser.id}`);
+        .put(`/rest/rooms/${randomId}/join/${testUser.id}`)
+        .set({ Authorization: `Bearer: ${token}` });
+      expect(result.status).to.equal(404);
+      expect(result.body).to.be.an('object');
+      expect(result.body.error).to.equal('A valid uuid roomId was passed in join room route but returned no match.');
+    });
+  });
+
+  // PUT leave room
+  describe('PUT /:roomId/leave/:userId', () => {
+    it('should return an object with the room and user ID removed from the join table', async () => {
+      const result = await agent
+        .put(`/rest/rooms/${testRooms[0].id}/leave/${testUser.id}`)
+        .set({ Authorization: `Bearer: ${token}` });
+      expect(result.status).to.equal(200);
+      expect(result.body).to.be.an('object');
+      expect(result.error).to.equal(false);
+      expect(result.body.room_id).to.match(uuidv4RegExp);
+      expect(result.body.user_id).to.match(uuidv4RegExp);
+    });
+
+    it('should return the correct status and error when passed an invalid roomId', async () => {
```

Comment:
So it may be more informative to specify what the correct status is here. 

For instance, `should return 400 and error message when passed an invalid roomId`

Same recommendation for the other test descriptions too